### PR TITLE
RPI2/Ubuntu: add uhubctl to restart USB Hub

### PR DESCRIPTION
**Pull request by rmackay9**
This PR corrects a few issues with the RPI2/Ubuntu scripts:

- install scripts executable permissions fixed
- adds uhubctl so that RPI4's USB ports are powered off/on to allow the T265 to start without user intervention
- simplifies the human readable install instructions to point users at the automated scripts
- removes swap file creation which allows the scripts to be run on an RPI4 with an 8GB SD card
- separates the install_librealsense and instasll_vision_to_mavros so they can be run separately
- adds clean-for-imaging script which is creates an image of the SD card when run on a PC (I think)

### CODE REVIEW

File: Common/Ubuntu/librealsense/stop_t265.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Stop t265 service
+
+if [ $(id -u) -ne 0 ]; then
+   echo >&2 "Must be run as root"
+   exit 1
+fi
+
+sudo systemctl stop t265
```
Comment (khancyr) [inline Common/Ubuntu/librealsense/stop_t265.sh]: nitpick, if you are root, you don't need sudo

File: RPI2/Ubuntu/clean-for-imaging [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+. config.env
+
+pushd ../../Common/Ubuntu/
+ ./clean-for-imaging
+popd
+
+rm -f /home/$NORMAL_USER/.ssh/authorized_keys
+
+dd if=/dev/zero of=$HOME/zeroes || true
```
Comment (khancyr) [inline RPI2/Ubuntu/clean-for-imaging:19]: dd if=/dev/zero of=$HOME/zeroes status=progress

File: Common/Ubuntu/librealsense/stop_t265.sh (hunk 1 above)
Comment (rmackay9) [reply Common/Ubuntu/librealsense/stop_t265.sh]: Thanks, fixed!

**Comment by rmackay9** [comment]
I think these scripts work now from start to finish.  I ran though a full install today but bumped into a few issues.  I've fixed those and I will try the full procedure again tomorrow.

**Comment by rmackay9** [comment]
I've tested these now from start to finish and it comes up with a working image in which the Copter appears in Africa without the user having to make any changes on the RPI4.  I'll go ahead and merge this soon and then hopefully get an image onto firmware.ardupilot.org.

File: RPI2/Ubuntu/clean-for-imaging (hunk 2 above)
Comment (rmackay9) [reply RPI2/Ubuntu/clean-for-imaging:19]: This part is from @peterbarker so I'll let him comment..

Comment (peterbarker) [reply RPI2/Ubuntu/clean-for-imaging:19]: @rmackay9 this just adds a comforter so you know things are moving along.

Wouldn't hurt.

**Pull request merged**